cartographer_ros/cartographer_ros/node.cc

```diff
@@ -220,6 +228,13 @@ void Node::PublishTrajectoryStates(const ::ros::WallTimerEvent& timer_event) {
   }
 }
 
+void Node::PublishTrajectoryNodesList(
+    const ::ros::WallTimerEvent& unused_timer_event) {
+  carto::common::MutexLocker lock(&mutex_);
+  trajectory_nodes_list_publisher_.publish(
```

Comment:
I added this in a956257eb0149ed8fbd8e5416accbdd626bc1246, feel free to revert it if you think it's unnecessary.